package.json

```diff
@@ -7,7 +7,7 @@
   "homepage": "https://nebenan.de/",
   "repository": "good-hood-gmbh/nebenan-helpers",
   "bugs": "https://github.com/good-hood-gmbh/nebenan-helpers/issues",
-  "version": "7.0.0",
+  "version": "7.0.1",
```

Comment:
no need to republish imo 
the compiled assets shouldn't have changed

Reply:
So shall I undo it or do we keep it?

Reply:
no point in publishing in my opinion - no difference between compiled assets from 7.0.0 and 7.0.1